packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -220,21 +222,21 @@ contract FastBridgeV2 is Admin, IFastBridgeV2 {
         bytes32 transactionId = keccak256(request);
         BridgeTransaction memory transaction = getBridgeTransaction(request);
 
-        // update bridge tx status if able to claim origin collateral
-        if (bridgeStatuses[transactionId] != BridgeStatus.RELAYER_PROVED) revert StatusIncorrect();
+        BridgeTxDetails storage _thisBridgeTxDetails = bridgeTxDetails[transactionId];
 
-        BridgeProof memory proof = bridgeProofs[transactionId];
+        // update bridge tx status if able to claim origin collateral
+        if (_thisBridgeTxDetails.status != BridgeStatus.RELAYER_PROVED) revert StatusIncorrect();
 
         // if "to" is zero addr, permissionlessly send funds to proven relayer
         if (to == address(0)) {
-            to = proof.relayer;
-        } else if (proof.relayer != msg.sender) {
+            to = _thisBridgeTxDetails.proofRelayer;
+        } else if (_thisBridgeTxDetails.proofRelayer != msg.sender) {
             revert SenderIncorrect();
         }
 
-        if (_timeSince(proof) <= DISPUTE_PERIOD) revert DisputePeriodNotPassed();
+        if (_timeSince(_thisBridgeTxDetails.proofBlockTimestamp) <= DISPUTE_PERIOD) revert DisputePeriodNotPassed();
 
-        bridgeStatuses[transactionId] = BridgeStatus.RELAYER_CLAIMED;
+        _thisBridgeTxDetails.status = BridgeStatus.RELAYER_CLAIMED;
```

Comment:
My reasoning against the storage syntax is that it gets harder to track that this is a storage write rather than a memory one, which gets a bit confusing review-wise and could result in a bad change if the struct variable was ever made `memory` again.

Reply:
my rationale for the storage reference was gas savings (and consistency, for the external views) -- but after using forge to test, the savings seemed minimal. switched to fully-qualified approach everywhere